content/items/rggen.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "RgGen"
+description: "Code generator for configuration and status registers"
+authors:
+  - Taichi Ishitani
+links:
+  gh?: "rggen/rggen"
+  docs?: "https://github.com/rggen/rggen/wiki"
+tags:
+  - Code generator
+  - CSR
+  - Configuration register
+  - SystemVerilog
+  - Verilog
+  - UVM reg
+categories:
+  - Tools
+licenses?:
+  - MIT
+active:
+  from: 2016
+talk: <ISSUE>
+---
+
+RgGen is a code generator tool to generate source files for configuration and status registers (CSR) from human readable register map specifications.
+
+RgGen has following features:
+
+* Generate following source files for CSR automatically from register map specifications
+    * SystemVerilog/Verilog RTL
```

Comment:
Is the VHDL output work in progress yet (https://github.com/rggen/rggen-vhdl-rtl) ?

Reply:
Yes.
This task is just started.
Currently, RgGen does not support VHDL output.

Reply:
I see. I recommend adding a brief description of this and maybe some other "just started" features. One of the mottos of this organisation/repo is to bring people together. If you specify what you want to achieve with the tool, others might be interested and pass by.

Reply:
OK. I will add.